issueCertificates.sh

```diff
@@ -10,14 +10,19 @@ WORKSPACE=${WORKSPACE:-"$(
 )"}
 
 OC_CLIENT="${WORKSPACE}/openshift-client/oc"
-KUBECONFIG_FILE="${WORKSPACE}/openshift-files/auth/kubeconfig"
+
+# Require KUBECONFIG to be define
```

Comment:
I'd be more comfortable with KUBECONFIG=${KUBECONFIG:-~/.kube/config} much like default for kubectl than requiring setting KUBECONFIG.